packages/renderer/src/lib/kube/details/EventsTable.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+import humanizeDuration from 'humanize-duration';
+import moment from 'moment';
+
+import type { EventUI } from '../../events/EventUI';
+
+interface Props {
+  events: EventUI[];
+}
+
+let { events }: Props = $props();
+
+$effect(() => {
+  events.sort((ev1, ev2) => ((ev1.lastTimestamp ?? Date.now()) < (ev2.lastTimestamp ?? Date.now()) ? -1 : 1));
+});
```

Comment:
Maybe we should not sort, but create a dedicated state, 

```ts
let sortedEvents: EventUI[] = $derived(events.toSorted((ev1, ev2) => ((ev1.lastTimestamp ?? Date.now()) < (ev2.lastTimestamp ?? Date.now()) ? -1 : 1)));
```

So we don't have to modify the props